src/c/helpers.h

```diff
@@ -33,7 +33,11 @@
 #endif
 #endif
 
-
+#ifdef __MINGW32__
+// Mingw does not have an sa_family_t according to:
+//   https://lists.gnu.org/archive/html/bug-gnulib/2009-05/msg00194.html
+typedef unsigned short sa_family_t;
```

Comment:
Can you try `ADDRESS_FAMILY` instead of `unsigned short`? I'm using [MSDN](https://docs.microsoft.com/en-us/windows/win32/winsock/sockaddr-2) as the reference.